app/models/job.rb

```diff
@@ -61,4 +61,9 @@ def send_email
     return unless status == "ERROR"
     ErrorMailer.email(self).deliver
   end
+
+  def create_event_dispatcher!
+    # If the event_receiver does not exist, it will be updated when the event_receiver is created
+    EventDispatcher.new(event_receiver: job_type.event_receiver, job: self, triggered: false)
```

Comment:
A job type `has_many` event receivers so I think we need to create a new dispatcher for each instead of just one

Reply:
Also is there a way to make false the default so we don't have to specify it?

Reply:
Oops, good catch 👍